packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -9,11 +9,40 @@ import Inject from '../../Inject';
 import getDefaultT from '../../translate';
 import TooltipTrigger from '../../TooltipTrigger';
 
-function TitleSubHeader({ title, iconId, loading, inProgress, editable, getComponent, ...rest }) {
+function TitleSubHeader({
+	title,
+	iconId,
+	loading,
+	inProgress,
+	editable,
+	getComponent,
+	onEdit,
+	onCancel,
+	onSubmit,
+	...rest
+}) {
+	const [edit, setEdit] = React.useState(false);
+	function onEditHook() {
```

Comment:
I would change these since I don't see any meaning of "hook" in them:
`onEditHook` -> `onEdit`
`onCancelHook` -> `onCancel`
`onSubmitHook` -> `onSubmit`

But due to name collision with the props, you could maybe do `handleEdit`, `handleCancel`, `handleSubmit`.